pulp_rpm/app/tasks/publishing.py

```diff
@@ -287,9 +287,15 @@ def get_checksum_type(name, checksum_types, default=CHECKSUM_TYPES.SHA256):
         checksum_types (dict): Checksum types for metadata and packages.
     Kwargs:
         default: The checksum type used if there is no specified nor original checksum type.
+        repo_versiion: Repo version that is being published.
     """
     original = checksum_types.get("original")
     metadata = checksum_types.get("metadata")
+    if repo_version:
+        package_qs = repo_version.content.filter(pulp_type="rpm.package")
```

Comment:
`pulp_type=Package.get_pulp_type()` is slightly preferred.

Reply:
ops, sure, it was copy/paste. i'll update